src/Control/Monad.purs

```diff
@@ -64,3 +65,22 @@ unlessM :: forall m. Monad m => m Boolean -> m Unit -> m Unit
 unlessM mb m =  do
   b <- mb
   unless b m
+
+-- | `ap` provides a default implementation of `(<*>)` for any `Monad`, without
+-- | using `(<*>)` as provided by the `Apply`-`Bind` superclass relationship.
```

Comment:
Bind -> Monad here too

Reply:
Why? Isn't this specifically referring to `Apply` and `Bind`? Not `Apply` and `Monad`?

Reply:
Because this function has a Monad constraint. It's saying that while you could implement `ap` by just writing `ap = apply` (since the function has a Monad constraint, and Apply is one of Monad's superclasses), we don't, because we want people to be able to write `apply = ap` in their Apply instances.

Reply:
Ah, thanks for clarifying.

Reply:
I've fixed that in latest commits.